lib/beaker/dsl/install_utils.rb

```diff
@@ -1,13 +1,14 @@
-[ 'foss', 'pe', 'puppet', 'ezbake', 'module' ].each do |lib|
+[ 'foss', 'puppet', 'ezbake', 'module' ].each do |lib|
     require "beaker/dsl/install_utils/#{lib}_utils"
 end
+require "beaker/dsl/install_utils/pe_defaults"
 
 module Beaker
   module DSL
     # Collection of installation methods and support
     module InstallUtils
       include DSL::InstallUtils::PuppetUtils
-      include DSL::InstallUtils::PEUtils
+      include DSL::InstallUtils::PEDefaults
```

Comment:
this is left in due to backwards compatibility.  It was tried to pull this out previously, but it got left in because tests wouldn't work without the paths being set properly on SUT setup.  This is still the case, and I've decided to leave this at this point rather than try to solve this issue at this time.